src/scss/mixins/_anchor.scss

```diff
@@ -0,0 +1,10 @@
+@mixin of-anchor() {
+  color: var(--of-link-color);
+  cursor: pointer;
+  text-decoration: var(--utrecht-link-text-decoration, underline);
+
+  &:hover {
+    color: var(--of-link-hover-color);
+    text-decoration: var(--utrecht-link-hover-text-decoration, none);
+  }
+}
```

Comment:
we actually completely want to avoid this, as it introduces OF-specific design tokens and doesn't allow us to lift on existing design tokens. See https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-link-design-tokens--page for a reference of existing design tokens.

In particular, we should move away from `--of-link-color` and `--of-link-hover-color` in favour of `--utrecht-link-color` and `--utrecht-link-hover-color`. This implies:

* update the backend links to have the `.utrecht-link` class name
* update the backend documentation to include the `utrecht.link.color` design token example
* add a datamigration in the backend to update the config design tokens, mapping from of-specific value to utrecht component
* update the e-mail/PDF templates to use the utrecht design token rather than our own


For the WYSIWYG fields, I would avoid the mixin structure in this PR, instead rather re-use the available styles with some sass features:

```scss
.#{prefix(body)} {
  @include modifier(wysiwyg) {
    @include wysiwyg;

    a {
      @extend .utrecht-link;
    }
  }
}
```
(see the [`@extend`](https://sass-lang.com/documentation/at-rules/extend) docs)

This will also require to add the `@import '@utrecht/components/dist/link/css/index';` into the `_anchor.scss` component (and then it can be removed from `nl-design-system-community.scss`)

Reply:
Backend PR: https://github.com/open-formulieren/open-forms/pull/2529